package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "graphql-codegen-typescript-mock-data",
-    "version": "1.2.1",
+    "version": "1.3.0",
```

Comment:
don't bump the version yourself, the CI will handle that

Reply:
Ah cool, reverted that change